Portfolio: setup and mobile version skeleton

Shewolf227 opened the pull request:
In this exercise I implemented the following:

1. Add `HTML` and `CSS` Linters, and Github Workflow
2. Create `index.html`
3. Add the toolbar and headline sections
4. Create `styles.css` and add styling to it for `index.html`

Shewolf227 committed:
Portfolio: setup and mobile version skeleton

Add HTML and CSS Linters, and Github Workflow

Create index.html
Add the toolbar and headline sections

Create CSS and add styling to it

Shewolf227 committed:
Update styles.css

Add hover and active effects on the social media icons

gemmen29 commented on css/styles.css:
```diff
@@ -0,0 +1,94 @@
+* {
```
Kindly try to match the Figma design for the mobile version you can achieve this by:
- Please remove the width for the `h3` that takes class `desc`.
- Kindly remove the height for the `h1` that takes class `hey-there`.
- Please add the `background-image` that exists in Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/158651824-593f9b1b-9f9e-40a4-8d07-9dd83b2b056e.png) | ![Template 3-Main-Mobile - Copy](https://user-images.githubusercontent.com/24703171/158652387-1e4c9f5a-0b9e-493c-8407-49d293461f9a.png) |

gemmen29 commented on index.html:
```diff
@@ -0,0 +1,38 @@
+      <div class="menu-icon"></div>
+      <div class="menu-icon"></div>
+      <div class="menu-icon"></div>
+    </section>
```
Kindly wrap this part with `nav` tag for semantic purposes, so it will be like:
```
<header>
   <nav>
       your code .......
   </nav>
</header>
```

gemmen29 commented on index.html:
```diff
@@ -0,0 +1,38 @@
+  <header class="header-container">
+    <section class="logo">
+      <p>Sonia Karungi</p>
+    </section>
```
Kindly change the `section` tag for the logo to be `a` tag and remove the `p` tag for semantic purposes.

gemmen29 commented on index.html:
```diff
@@ -0,0 +1,38 @@
+    <h2 class="soft-dev">I am a Software Developer</h2>
+    <h3 class="desc">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you
+      see and have a project you need coded, don’t hestiate to contact me.
+    </h3>
```
Kindly change the `h3` tag to be `p` tag for semantic purposes.

gemmen29 commented on index.html:
```diff
@@ -0,0 +1,38 @@
+    <h3 class="desc">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you
+      see and have a project you need coded, don’t hestiate to contact me.
+    </h3>
+    <section class="contact-icons">
+      <img class="item-icon" src="img/social-media-icons/github.png" alt="GitHub Logo">
+      <img class="item-icon" src="img/social-media-icons/linkedin.png" alt="linkedin Logo">
+      <img class="item-icon" src="img/social-media-icons/twitter.png"  alt="Twitter Logo">
+    </section>
```
Kindly change the `section` tag to be `ul` tag and wrap each image with a `li` tag for semantic purposes.

gemmen29 reviewed (changes requested):
Hi @SoniaKarungi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shewolf227 committed:
feedback implementation

1.  remove the width for the h3 that takes class desc
2. remove the height for the h1 that takes class hey-there
3. add the background-image that exists in Figma design
4. wrap nav-menu section with nav tag
5. change the section tag for the logo to be a tag and remove the p tag
6. change the h3 tag to be p tag
7. change the section tag to be ul tag and wrap each image with a li

Shewolf227 committed:
Solve linting issues

Shewolf227 committed:
Deleted unnecessary styles file

AnselemOdims commented on index.html:
```diff
@@ -0,0 +1,38 @@
+      see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <ul class="contact-icons">
+      <li><img class="item-icon" src="img/social-media-icons/github.png" alt="GitHub Logo"></li>
+      <li><img class="item-icon" src="img/social-media-icons/linkedin.png" alt="linkedin Logo"></li>
+      <li><img class="item-icon" src="img/social-media-icons/twitter.png" alt="Twitter Logo"></li>
```
- [ ] [OPTIONAL] Please make sure you use all the icons as specified in the original design

AnselemOdims reviewed (approved):
### Hi  @SoniaKarungi 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
